force-app/main/default/lwc/crmOppgaveRecordSyncer/crmOppgaveRecordSyncer.js

```diff
@@ -14,13 +14,13 @@ export default class CrmOppgaveRecordSyncer extends LightningElement {
     wiredRecordInfo({ error, data }) {
         if (data) {
             const oppgaveRef = getFieldValue(data, EXTERNAL_REF_FIELD);
-            syncOppgaveById(oppgaveRef)
+            Promise.all([syncOppgaveById(oppgaveRef), syncOppgaveOppfolgingById(oppgaveRef)])
                 .then(() => {
-                    //Synced
-                    getRecordNotifyChange([{ recordId: this.recordId }]);
+                    // Both syncs completed successfully
+                    notifyRecordUpdateAvailable([{ recordId: this.recordId }]);
```

Comment:
Burde vi ikke notify hvis en av de feiler men den andre ikke gjør det? Prøv å bruk Promise.allSettled og sjekk status for hver https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled (Aldri gjort det selv, men det blir nok bedre.)

Reply:
Sure. Går omtrent ikke an at promisene ikke resolver/rejecter mtp. at vi har all logikk i try catch, men skader ikke å bruke allSettled 😄